docs/decisions/0004-issue-tracking.rst

```diff
@@ -0,0 +1,61 @@
+===================================
+Documenting and tracking BTR Issues
+===================================
+
+-------
+Status
+-------
+
+Draft
+
+-------
+Context
+-------
+
+With the move to a Github Project to manage tasks, the BTR needs to 
+define a clear process for documenting and managing issues.
+
+This document aims to outline a set of guidelines to manage the work of
+the BTR group on Github. These guidelines that are expected to evolve 
+to meet any particular need that arises.
+
+Last update: 2021-03-15
+
+---------
+Decisions
+---------
+
+#. A piece of work should be created as a github issue. Github projects <support notes", 
+   however these are too basic and don't support the additional information issues allows 
+   (for example, an assignee).
+
+#. To assess and discuss each issue, we need to know:
+
+   #. The type of issue: is it a bug? is it a gap in documentation?
+        - Tagged with <type:'issue-type'>
+        - Issue types: bug, enhancement, question, btr-task, security.
+   #. The release that is affected. 
+        - Tagged with <affects:'named-released'>
```

Comment:
I'm for this, but I don't think *every* issue needs an "affects:release".  For instance, an "enhancement" can't really affect anything until it exists.

I actually think that "affects" should be used almost exclusively for bugs.  Everything else should just have a milestone.

Reply:
Yes, it makes sense.